run.js

```diff
@@ -41,6 +44,23 @@ function promptWithDefault (question, defaultValue) {
   })
 }
 
+function nyplSourceAwareArrayChunks (arr, chunkSize = 25) {
```

Comment:
can you add comments describing whats happening here?

Reply:
Added function documentation.